### PR TITLE
feat: support imgix as an image service, and report perf numbers

### PR DESCRIPTION
**Pull request by mzikherman**
This adds support for imgix as an image resizer, which I've added to the staging and production ENV.

It also moves our lambda URL into an ENV var (already added to staging and production).

Additionally, it tweaks the performance reporting to allow images from any of the supported image resize domains to be supported.

### CODE REVIEW

File: src/Utils/Hooks/useImagePerformanceObserver.ts [hunk 1]
```diff
@@ -28,8 +34,8 @@ export const useImagePerformanceObserver = () => {
             entry.initiatorType !== "img" ||
             // Ensure they are uncached
             entry.transferSize === 0 ||
-            // Ensure they are Gemini images
-            !entry.name.includes(GEMINI_CLOUDFRONT_URL) ||
+            // Ensure they come from image domains we want to track
+            !TRACKING_URL_ALLOWLIST.some(url => entry.name.includes(url)) ||
```
Comment (mzikherman) [inline src/Utils/Hooks/useImagePerformanceObserver.ts:38]: `includes` is likely a bit permissive over `startsWith` but /shrug

**Review by dzucconi — APPROVED** [review]
🙏

File: src/Utils/Hooks/useImagePerformanceObserver.ts (hunk 1 above)
Comment (joeyAghion) [reply src/Utils/Hooks/useImagePerformanceObserver.ts:38]: OHHH

**Comment by mzikherman** [comment]
Gonna try to merge, even tho the test timeout is plaguing it.

**Pull request merged**